spec/factories/users.rb

```diff
@@ -0,0 +1,20 @@
+# include ActionDispatch::TestProcess
+FactoryGirl.define do
```

Comment:
Using FactoryGirl has made your tests much cleaner than they would be otherwise ;-) 
Have a look at this though: https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md#sequences